index.html

```diff
@@ -7,23 +7,24 @@
 <tr><td style=" padding : 20px; width:175px">
 <img src="/images/naren.jpg" style="border: 1px black; "></td>
 <td><h2>About</h2>
-  I am a software engineer at Google. I am interested in systems and machine learning. This July (2015), I got my Masters in Computer Science from University of California, San Diego. I was advised by <a href="http://weibel.ucsd.edu/">Dr. Nadir Weibel</a>.
```

Comment:
Testing PR and review capabilities in Github. Did you try the following change?

`import time
time.time()`